package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.3",
   "license": "MIT",
   "scripts": {
-    "lint": "eslint . --ext .ts --fix",
+    "lint": "yarn lint:nofix --fix",
+    "lint:nofix": "eslint . --ext .ts",
```

Comment:
nit
```suggestion
    "lint:fix": "yarn lint --fix",
    "lint": "eslint . --ext .ts",
```

Reply:
I also wished that could happen :P

This was kept like that because most of the project was using `yarn lint` with --fix by default. This is a technical debt, a similar PR with this fix was rejected in the past.

Maybe a future organization wide PR will fix that, on the DX side